shell/src/main/java/alluxio/cli/ValidateHdfsMount.java

```diff
@@ -183,27 +129,18 @@ public static void main(String[] args) throws Exception {
       LOG.debug("Options from cmdline: {}", properties);
     }
 
-    // Run validateEnv
-    List<ValidationUtils.TaskResult> results = validateUfs(ufsPath, ufsConf);
-
-    // Run runUfsTests
-    if (ufsConf.isReadOnly()) {
-      LOG.debug("Ufs operations are skipped because the path is readonly.");
-      results.add(new ValidationUtils.TaskResult(ValidationUtils.State.SKIPPED,
-              UnderFileSystemContractTest.TASK_NAME,
-              String.format("UFS path %s is readonly, skipped UFS operation tests.", ufsPath),
-              ""));
-    } else {
-      results.add(runUfsTests(ufsPath, new InstancedConfiguration(ufsConf)));
-    }
+    ValidationToolRegistry registry
+            = new ValidationToolRegistry(new InstancedConfiguration(ConfigurationUtils.defaults()));
+    // Load hdfs validation tool from alluxio lib directory
+    registry.refresh();
```

Comment:
Not for this PR, but looks like we refresh the lib load every time we run hmsValidatation or hdfsValidation. It may not be a big problem for now, but looks like I need to consider addressing. @madanadit Will it be better to create the registry and refresh for once one

Reply:
if this behavior is the same on the server side invocation, this can be a bigger issue. the client jvm is short-lived, so its less of an issue